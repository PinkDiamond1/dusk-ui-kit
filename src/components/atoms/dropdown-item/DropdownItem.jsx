/* @flow */
import * as React from 'react'
import BootstrapDropdownItem from 'react-bootstrap/DropdownItem'

type Props = {
  children: React.Node,
  href?: string,
}

const DropdownItem = (props: Props): React.Element<*> => {
  const { children, href } = props
  // Extend defaults here...
  return <BootstrapDropdownItem href={href}>{children}</BootstrapDropdownItem>
}

DropdownItem.defaultProps = {
  href: '#',
}

export default DropdownItem
