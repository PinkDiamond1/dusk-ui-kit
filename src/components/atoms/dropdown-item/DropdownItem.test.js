import React from 'react'
import renderer from 'react-test-renderer'

import DropdownItem from './DropdownItem'

it('test to see if the DropdownItem renders correctly', () => {
  const tree = renderer.create(<DropdownItem>Smoke Test</DropdownItem>).toJSON()
  expect(tree).toMatchSnapshot()
})
