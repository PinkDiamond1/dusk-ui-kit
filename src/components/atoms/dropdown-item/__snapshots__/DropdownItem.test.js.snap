// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if the DropdownItem renders correctly 1`] = `
<a
  className="dropdown-item"
  href="#"
  onClick={[Function]}
  onKeyDown={[Function]}
  role="button"
>
  Smoke Test
</a>
`;
